velox/connectors/hive/storage_adapters/s3fs/S3Util.h

```diff
@@ -48,6 +48,14 @@ inline void bucketAndKeyFromS3Path(
   key = path.substr(firstSep + 1);
 }
 
+inline std::string s3URI(const char* bucket) {
+  return std::string(kS3Scheme) + bucket;
+}
+
+inline std::string s3URI(const char* bucket, const char* key) {
```

Comment:
Why is this function taking bucket and key as char* and not std::string. Another function in this file, bucketAndKeyFromS3Path, takes these as std::string's.

Reply:
I will unify the API. My oversight.